### PR TITLE
Specific settings : Handle NaN values.

### PR DESCRIPTION
**Pull request by AAJELLAL**

### CODE REVIEW

File: src/components/dialogs/parameters/load-flow-parameters.js [hunk 1]
```diff
@@ -318,6 +318,16 @@ const SpecificLoadFlowParameters = ({
         return extractDefaultMap(specificParamsDescription);
     }, [specificParamsDescription]);
 
+    // change object's NaN values into null
+    const getObjectWithoutNanValuse = useCallback((initialObject) => {
```
Comment (FranckLecuyer) [inline src/components/dialogs/parameters/load-flow-parameters.js:322]: Typo: getObjectWithoutNanValues

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/components/dialogs/parameters/load-flow-parameters.js (hunk 1 above)
Comment (AAJELLAL) [reply src/components/dialogs/parameters/load-flow-parameters.js]: Done

**Review by FranckLecuyer — APPROVED** [review]

File: src/components/dialogs/parameters/load-flow-parameters.js [hunk 2]
```diff
@@ -318,6 +318,16 @@ const SpecificLoadFlowParameters = ({
         return extractDefaultMap(specificParamsDescription);
     }, [specificParamsDescription]);
 
+    // change object's NaN values into null
+    const getObjectWithoutNanValues = useCallback((initialObject) => {
+        initialObject &&
+            Object.keys(initialObject)?.map((key) =>
+                Number.isNaN(initialObject[key])
+                    ? (initialObject[key] = null)
+                    : initialObject[key]
```
Comment (ayolab) [inline src/components/dialogs/parameters/load-flow-parameters.js:327]:
```suggestion
                (initialObject[key] = Number.isNaN(initialObject[key])
                    ? null
                    : initialObject[key])
```

File: src/components/dialogs/parameters/load-flow-parameters.js [hunk 3]
```diff
@@ -331,17 +341,29 @@ const SpecificLoadFlowParameters = ({
...
+
             commitLFParameter(toSend);
         },
-        [commitLFParameter, currentProvider, defaultValues, lfParams]
+        [
+            commitLFParameter,
+            currentProvider,
+            defaultValues,
+            lfParams,
+            getObjectWithoutNanValuse,
```
Comment (ayolab) [inline src/components/dialogs/parameters/load-flow-parameters.js:363]:
```suggestion
            getObjectWithoutNanValues,
```

**Review by ayolab — APPROVED** [review]

File: src/components/dialogs/parameters/load-flow-parameters.js (hunk 2 above)
Comment (AAJELLAL) [reply src/components/dialogs/parameters/load-flow-parameters.js]: Done

**Pull request merged**